README.rst

```diff
@@ -205,3 +205,36 @@ Please see ``--help`` for more options.
 ::
 
 $ python host_os.py build-iso --help
+
+How to contribute a new package to Host OS
+------------------------------------------
+
+Create a new directory at https://github.com/open-power-host-os/versions repository named <package_name>.
+
+Under this new directory: 
+
+1. Create a metadata file <package_name>.yaml. It will contain most of the information required to build the package. The package metadata (yaml) file format is at https://github.com/open-power-host-os/versions/package_metadata_yaml_format.yaml
+2. Create subdirectory "centOS/7", current "Linux Distribution Name"/"Linux Distribution Version", respectively
```

Comment:
=> Create two subdirectories named <distro_name> and <distro_name>/distro_version>, eg CentOS and CentOS/7